### PR TITLE
Address feedback from #147

### PR DESCRIPTION
**Pull request by changhiskhan**
isort'd everything so there are a bunch more changes.

### CODE REVIEW

File: python/rikai/numpy.py [hunk 1]
```diff
@@ -34,10 +34,10 @@
 # Third Party
 import numpy as np
 
-# Rikai
-from rikai.spark.types import NDArrayType
 from rikai.mixin import ToNumpy
```
Comment (eddyxu) [inline python/rikai/numpy.py:37]: Move `rikai` package together with another round of isort?

File: python/rikai/spark/conf.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# for parquet.block.size when caching during df -> Dataset conversion
+CONF_PARQUET_BLOCK_SIZE = "parquet.block.size"
+DEFAULT_ROW_GROUP_SIZE_BYTES = 32 * 1024 * 1024
+
+# config name for the cache root during df -> Dataset conversion
+CONF_SPARK_RIKAI_CACHEURI = "spark.rikai.cacheUri"
```
Comment (eddyxu) [inline python/rikai/spark/conf.py:20]: We need another round of clean up for the configuration namespaces for rikai keys.

File: python/rikai/spark/sql/codegen/fs.py [hunk 3]
```diff
@@ -13,18 +13,18 @@
 #  limitations under the License.
 
 import secrets
-from typing import Callable, IO, Any, Dict, Mapping, Optional, Union
+from typing import IO, Any, Callable, Dict, Mapping, Optional, Union
```
Comment (eddyxu) [inline python/rikai/spark/sql/codegen/fs.py:16]: it is weird that IO is before Any

File: python/rikai/spark/sql/generated/RikaiModelSchemaLexer.py [hunk 4]
```diff
@@ -1,9 +1,9 @@
 # Generated from src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiModelSchema.g4 by ANTLR 4.9.1
-from antlr4 import *
+import sys
```
Comment (eddyxu) [inline python/rikai/spark/sql/generated/RikaiModelSchemaLexer.py:2]: Can we set `isort` to ignore this directory?

Those changes will be override next time we generate the code.

File: python/tests/conftest.py [hunk 5]
```diff
@@ -16,8 +16,8 @@
 
 # Third Party
 import pytest
-from torch.utils.data import DataLoader  # Prevent DataLoader hangs
 from pyspark.sql import SparkSession
+from torch.utils.data import DataLoader  # Prevent DataLoader hangs
```
Comment (eddyxu) [inline python/tests/conftest.py:20]: Last time I did this, the DataLoader hangs.  Not sure whether it is still the case this time.

**Review by eddyxu — APPROVED** [review]
small nits

File: python/rikai/spark/sql/codegen/fs.py (hunk 3 above)
Comment (changhiskhan) [reply python/rikai/spark/sql/codegen/fs.py:16]: i'm turning off `order_by_type` (isort thinks IO is a constant)

**Pull request merged**